src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -1673,5 +1682,27 @@ internal override bool HasAsyncMethodBuilderAttribute(out TypeSymbol builderArgu
             builderArgument = _containingType.ContainingPEModule.TryDecodeAttributeWithTypeArgument(this.Handle, AttributeDescription.AsyncMethodBuilderAttribute);
             return builderArgument is not null;
         }
+
+        internal override int? TryGetOverloadResolutionPriority()
+        {
+            if (!_packedFlags.IsOverloadResolutionPriorityPopulated)
+            {
+                if (_containingType.ContainingPEModule.Module.TryGetOverloadResolutionPriorityValue(_handle, out int priority))
+                {
```

Comment:
We can't easily do that for accessors (and I have tests demonstrating the behavior), since we don't track, at the PEMethodSymbol layer, if this method is an accessor for some property or event. I instead opted to just read them, and not care whether the language may not permit them to be declared. We could certainly optimize the override/destructor/non-indexer properties, but it didn't seem like it would be worth the effort to me. If a tree falls in the forest but no one is there to hear it, did it make a sound?